universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -16,10 +17,22 @@ public void put(String path, String key, String value) {
     this.jsonData.put(path, key, value);
   }
 
+  public void put(String path, String key, Object value) {
+    this.jsonData.put(path, key, value);
+  }
+
   public <T> T read(String path, Class<T> type) {
     return this.jsonData.read(path, type);
   }
 
+  public Instant getCreatedTime() {
```

Comment:
suggestion (non-blocking): It's probably time to start adding unit tests for ApplicantData assuming `getCreatedTime` and `setCreatedTime` will stick around.

Reply:
Good idea - done, and there was a bug here so it's good to have written it.